portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -90,6 +91,17 @@ async function deleteData() {
   getData();
 }
 
+/**
+ * Fetches image upload URL from blobstore-upload-url servlet and sets
+ * URL as action in {@code comment-image-form}.
+ */
+async function getSetBlobUrl() {
```

Comment:
Why is this method called getSetBlobUrl? Typically get is appended for when you are returning something and set is for when you are setting a value. Here you aren't doing either. Maybe rename populateBlobUrl?

Reply:
I called it getSetBlobUrl because in the method I retrieve the BlobUrl and set it as the action in the HTML. populateBlobUrl sounds like a better name though